src/main/java/com/adyen/model/PaymentResult.java

```diff
@@ -68,38 +68,30 @@
      * the result of the payment
      */
     public enum ResultCodeEnum {
+        @SerializedName("AuthenticationFinished")
+        AUTHENTICATIONFINISHED("AuthenticationFinished"),
+        @SerializedName("AuthenticationNotRequired")
+        AUTHENTICATIONNOTREQUIRED("AuthenticationNotRequired"),
```

Comment:
AuthenticationNotRequired is not part of api v51

Reply:
Removed